blocks_vertical/procedures.js

```diff
@@ -799,24 +789,25 @@ Blockly.Blocks['procedures_call'] = {
     this.procCode_ = '';
     this.argumentIds_ = [];
     this.warp_ = false;
+
+    // Shared.
+    this.getProcCode = getProcCode.bind(this);
+    this.removeAllInputs_ = removeAllInputs_.bind(this);
+    this.disconnectOldBlocks_ = disconnectOldBlocks_.bind(this);
+    this.deleteShadows_ = deleteShadows_.bind(this);
+    this.createAllInputs_ = createAllInputs_.bind(this);
+    this.updateDisplay_ = updateDisplay_.bind(this);
+
+    // Exist on all three blocks, but have different implementations.
+    this.mutationToDom = callerMutationToDom.bind(this);
+    this.domToMutation = callerDomToMutation.bind(this);
+    this.populateArgument_ = populateArgumentOnCaller_.bind(this);
+    this.addProcedureLabel_ = addLabelField_.bind(this);
+
+    // Only exists on the external caller.
```

Comment:
I know this isn't your comment but I'm confused why this says it only exists on the caller when this is a def block.

Reply:
This is on the caller, Github's diff collapsing makes it look like it's on the def.